tuf-notary/tuf-repository.go

```diff
@@ -14,5 +14,47 @@ func Init(repository string) error {
 	}
 
 	//not using consistent snapshots
-	return repo.Init(false)
+	err = repo.Init(false)
```

Comment:
What is the point of this file, sorry? Totally not clear.

Reply:
it sets up the TUF repository locally.